### ISSUE TITLE
npm install do not include icons/

### ISSUE THREAD

**Issue by Mombuyish** [bartholomej/material-scrolltop#5]
when I use
```
$ npm install material-scrolltop --save-dev or yarn
```
I found the package do not include src/icons/top-arrow.svg, so I compiled by webpack got the error.
but I use bower it's work fine.

my temporary solution:
copy the SVG file into the folder and compile it.

**Comment by bartholomej** [comment]
Thanks @Mombuyish! Should be fixed in version `v1.0.1.` Try to upgrade and let me know ;)

BTW: May I ask you to show us a little bit from your implementation and how did you integrate `material-scrolltop` into your site? I'll add it into readme for everyone ;) PR welcome ;)

**Comment by Mombuyish** [comment]
It's works! Thank for your works!

I'm using Laravel mix to integrate into my site, it's base on webpack.
Here is my installation step: 

1. Installation
``` bash
$ npm install material-scrolltop --save-dev
```

2. require js.
``` js
require('material-scrolltop');
```

3. import scss
``` scss
@import "~material-scrolltop/src/material-scrolltop";
```

4. compiled to app.js and app.css by webpack
``` bash
$ npm run node_modules/webpack/bin/webpack.js --progress --hide-modules
```

it's done.
Maybe using basic webpack have some different, but also same as.

Again, thanks. Nice works.
